### ISSUE TITLE
Hi, I get 0 after training

### ISSUE THREAD

**Issue by venusafroid** [aykutfirat/pyTorchTree#1]
Hi,

Here is what I got:

Reading and parsing trees
Epoch 0
100% |##########################################################| Time: 0:17:03
100% |##########################################################| Time: 0:00:09

Validation All-nodes accuracy:tensor(0)(best:0.0)
Validation Root accuracy:tensor(1, dtype=torch.uint8)(best:tensor(1, dtype=torch.uint8))

I will be very grateful If I can get your reply.
